components/App.js

```diff
@@ -29,8 +29,8 @@ var App = React.createClass({
     render: function () {
         return (
             <div className='app'>
-                ContactForm={contact: contactForm};
-                Contacts = {items: contacts};
+                ContactForm={contact= contactForm};
+                Contacts = {items= contacts};
 
```

Comment:
To nie jest poprawna składnia.  Komponent traktujesz w JSX jak każdy inny element html, tyle tylko, że pisany z dużej litery
  `<Contacts items={contacts} />`